web/src/components/Unauth.js

```diff
@@ -130,6 +130,15 @@ const Button = styled.button`
   margin-bottom: 15px;
 `;
 
+const QRCodeCloseButton = styled.button`
+  padding: 8px 15px;
+  font-size: 6px;
+  position: absolute;
+  right: 20px;
+  border-radius: 20px;
+  margin-bottom: 15px;
+`;
```

Comment:
💡 if  want more responsive to window size , you can try relative unit like `em` instead absolute unit `px`